### PR TITLE
Add worklog functionality to jira module

### LINKED ISSUE #6209: Add worklog comment for Jira module
### Summary

In the `jira` module we have the ability to create the Jira and estimate the times.

This is achieved by the `/worklog` api similar to what `comment` does:

* https://docs.atlassian.com/software/jira/docs/api/REST/7.3.3/?_ga=2.174542626.1666439307.1519293627-1080493640.1514451808#api/2/issue-updateWorklog

This is really interesting if you share times of user vs automation.

### Issue Type

Feature Idea

### Component Name

community.general.jira

### Additional Information

```yaml (paste below)
- name: Worklog on issue
  community.general.jira:
    uri: '{{ server }}'
    username: '{{ user }}'
    password: '{{ pass }}'
    issue: '{{ issue.meta.key }}'
    operation: worklog
    comment: A worklog added by Ansible
    fields:
      timeSpentSeconds: 12000
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by hugoprudente**
##### SUMMARY
This adds the functionality to jira be able to add worklog comments.

Fixes #6209

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
community.general.jira

```paste below
- name: Worklog on issue
  community.general.jira:
    uri: '{{ server }}'
    username: '{{ user }}'
    password: '{{ pass }}'
    issue: '{{ issue.meta.key }}'
    operation: worklog
    comment: A worklog added by Ansible
    fields:
      timeSpentSeconds: 12000
```

### CODE REVIEW

**Comment by hugoprudente** [comment]
Last commit adds the integration test that I forgot. Please let me know if something else is missing.

File: changelogs/fragments/6210-add-worklog-functionality-to-jira.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - jira module, add worklog functionality (https://github.com/ansible-collections/community.general/issues/6209, https://github.com/ansible-collections/community.general/pull/6210).
```
Comment (felixfontein) [inline changelogs/fragments/6210-add-worklog-functionality-to-jira.yml:2]:
```suggestion
  - jira - add worklog functionality (https://github.com/ansible-collections/community.general/issues/6209, https://github.com/ansible-collections/community.general/pull/6210).
```

File: plugins/modules/jira.py [hunk 2]
```diff
@@ -40,7 +41,7 @@
     type: str
     required: true
     aliases: [ command ]
-    choices: [ attach, comment, create, edit, fetch, link, search, transition, update ]
+    choices: [ attach, comment, create, edit, fetch, link, search, transition, update, worklog ]
     description:
       - The operation to perform.
```
Comment (felixfontein) [inline plugins/modules/jira.py:46]:
```suggestion
      - The operation to perform.
      - C(worklog) was added in community.genereal 6.5.0.
```

File: plugins/modules/jira.py [hunk 3]
```diff
@@ -197,7 +197,7 @@
   validate_certs:
     required: false
     description:
-      - Require valid SSL certificates (set to C(false) if you'd like to use self-signed certificates)
+      - Require valid SSL certificates (set to `false` if you'd like to use self-signed certificates)
```
Comment (felixfontein) [inline plugins/modules/jira.py:200]: Please revert this, using backticks is wrong, you must use `C()`.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! I've added some first comments below.

**Comment by hugoprudente** [comment]
Hi @felixfontein addressed all the comments thanks for the initial review.

**Review by felixfontein — COMMENTED** [review]
Looks good to me (without knowing or using jira). If nobody objects, I'll merge this for 6.5.0.

**Comment by felixfontein** [comment]
@hugoprudente thanks for your contribution!

**Pull request merged**